quiz-pages/badminton-trivia-quiz/rules.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rules | Badminton Trivia</title>
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="../../styles/rules.css">
+    <link rel="stylesheet" href="https://shuttleui.netlify.app/components/components-styles/shuttleUI-main.css">
+    <script src="https://kit.fontawesome.com/2352d5969b.js" crossorigin="anonymous" defer></script>
+</head>
+<body>
+    
+        <nav class="navbar flex-row">
+            <a class="navbar-brand" href="../../index.html">
+                <img class="navbar-logo" 
+                src="../../assets/site-logo.png" alt="site logo">
+            </a>
+    
+            <ul class="navbar-menu list-unstyled flex-row">
+                <li class="list-inline">
+                    <a class=" menu-link" href="../../index.html">   
+                        <i class="fas fa-home"></i> Home
+                    </a>
+                </li>
+                <li class="list-inline">
+                    <a class=" menu-link" href=""> 
+                        <i class="fas fa-sign-in-alt"></i> Login
+                    </a>
+                </li>             
+            </ul>
+        </nav>
+
+        <main class="main align-items-center">
+            <div class="card-rules card-shadow-alternate">
+                <div class="card-header">
+                    <h1 class="text-center  rules-heading">Rules</h1>
+                </div>
+               
+                <div class="card-body align-items-center">
+                    <ul>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>There will be 5 quesitons.</li>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>Each question will have one correct answer.</li>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>Correct answer will provide 5 points.</li>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>Incorrect answer will deduct 1 point.</li>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>To unlock new level you have to score 75%.</li>
+                        
+                        <li class="list-unstyled"></li>
```

Comment:
Is this required here? The last li tag

Reply:
No, forgot to remove it. Thanks for pointing this out.